assets/javascripts/discourse/initializers/journal-topic.js

```diff
@@ -127,138 +126,32 @@ export default {
-      //       h(
-      //         "li",
-      //         h("a", { attributes: { href: lastEntryUrl } }, [
-      //           h("h4", I18n.t(`last_entry_lowercase`)),
-      //           h("div.topic-map-post.last-entry", [
-      //             avatarFor("tiny", {
-      //               username: topic.journal_author.username,
-      //               template: topic.journal_author.avatar_template,
-      //               name: topic.journal_author.name
-      //             }),
-      //             dateNode(attrs.lastPostAt)
-      //           ])
-      //         ])
-      //       )
-      //     );
-
-      //     contents.push(
-      //       h("li", [
-      //         numberNode(topic.entry_count),
-      //         h(
-      //           "h4",
-      //           I18n.t(`entry_lowercase`, { count: topic.entry_count })
-      //         )
-      //       ])
-      //     );
-
-      //     contents.push(
-      //       h("li", [
-      //         numberNode(topic.comment_count),
-      //         h(
-      //           "h4",
-      //           I18n.t(`comment_lowercase`, { count: topic.comment_count })
-      //         )
-      //       ])
-      //     );
-
-      //     contents.push(
-      //       h("li.secondary", [
-      //         numberNode(attrs.topicViews, { className: attrs.topicViewsHeat }),
-      //         h("h4", I18n.t("views_lowercase", { count: attrs.topicViews }))
-      //       ])
-      //     );
-
-      //     if (attrs.topicLikeCount) {
-      //       contents.push(
-      //         h("li.secondary", [
-      //           numberNode(attrs.topicLikeCount),
-      //           h("h4", I18n.t("likes_lowercase", { count: attrs.topicLikeCount }))
-      //         ])
-      //       );
-      //     }
-
-      //     if (attrs.topicLinkLength > 0) {
-      //       contents.push(
-      //         h("li.secondary", [
-      //           numberNode(attrs.topicLinkLength),
-      //           h("h4", I18n.t("links_lowercase", { count: attrs.topicLinkLength }))
-      //         ])
-      //       );
-      //     }
-
-      //     if (
-      //       state.collapsed &&
-      //       attrs.topicPostsCount > 2 &&
-      //       attrs.participants.length > 0
-      //     ) {
-      //       const participants = renderParticipants.call(
-      //         this,
-      //         attrs.userFilters,
-      //         attrs.participants.slice(0, 3)
-      //       );
-      //       contents.push(h("li.avatars", participants));
-      //     }
-
-      //     const nav = h(
-      //       "nav.buttons",
-      //       this.attach("button", {
-      //         title: "topic.toggle_information",
-      //         icon: state.collapsed ? "chevron-down" : "chevron-up",
-      //         action: "toggleMap",
-      //         className: "btn"
-      //       })
-      //     );
-
-      //     return [nav, h("ul.clearfix", contents)];
-      //   }
-      // });
+      api.reopenWidget("topic-map", {
```

Comment:
Just noting - we're are currently halfway through modernising the topic-map in core, so this call will also break in the next couple of weeks, sorry!

Very happy to add Plugin Outlets (wrapper or otherwise) into core so that you can implement the same thing 100% in components. Feel free to mention me on a PR, or in a Meta `#dev` topic if you'd like to discuss ideas.

Reply:
Thanks David!  This would be a great example use case for future occasions too.

Reply:
Good to know, @davidtaylorhq! 

@merefield I'll think about what outlets would be needed and comment on this PR 👍🏼